providers/frame/FrameBinProvider.ts

```diff
@@ -43,10 +43,10 @@ export abstract class FrameBinProvider extends FrameProxyProvider {
       args,
       stdout: "piped",
       stderr: "piped",
+      signal: this.env.signal,
```

Comment:
not sure if this requires a `.bind(this)`

Reply:
It's not a function, so no